### DISCUSSION TITLE
Should outfit scans reveal spoilers?

### DISCUSSION THREAD

**Discussion by opusforlife2** [endless-sky/endless-sky#10708] [Lore]
For example, author ships often carry powerful late-game/end-game outfits (Electron Beam/Dual Sunbeam Turret/alien outfits in general) that will be revealed to the player much later.

Maybe the items in the scan list could just say "Unknown Power Outfit", or "Unknown Wanderer Turret"?

This wouldn't make sense for unobtainable outfits, though, such as Quarg ones. Knowledge of them is irrelevant to the lore.

**Comment by TheGiraffe3** [comment]
https://github.com/endless-sky/endless-sky/pull/8532
https://github.com/endless-sky/endless-sky/pull/8531#issuecomment-1475379970:
> The problem with that reasoning is that realism has never really been our aim.

Doesn't look like that's something which is wanted.

**Comment by Hecter94** [comment]
I'd say it's fine. Just knowing the name of the outfit doesn't really spoil anything. 

You might see something named "Skylance" or "Sunbeam", but you don't know anything about it. You don't even know what type of outfit/weapon it is, who it belongs to, or even that it's alien technology. 
If anything, it's just a bit of a preview to build intrigue.

**Comment by opusforlife2** [comment]
Suppose the player knows of Sunbeams, but hasn't played the Wanderer campaign. Then seeing a Dual Sunbeam on a scan gives a nice hint as to what the player may look forward to if they interact with the Wanderers further.

Maybe if the player has access to a particular species' outfits, the outfit scan could just add their name to the end of new outfits. Like `Electron Beam (Human)`. It would make sense in-lore that an unknown weapon could still be identified as roughly human-made or Hai-made based on the tech that shows up in the scan, once the player has visited that species' outfitter/has the relevant license.

**Comment by esksp** [comment]
I agree: **Unknown outfits from strange alien civilizations (that includes "authors"), should have a special "unknown equipment" caption**.

**I especially object to the extensive list of outfits you see in the map when choosing "Shipyards" and "Outfitters"**: Barely have you just met a civilization, and you already get a full list of _all_ their technologies, complete with places where you (potentially) could obtain them. This is ridiculous! You shouldn't even know what those people call themselves until somebody tells you so...

Example (in my game): Kor Mereti and Kor Sestor. I have stumbled upon them, occasionally fought them (hard not to), but I don't know (or at least I'm not supposed to know) anything specific about them, except they might be vaguely Korath-related. And yet the map view gladly gives me a full list of their technologies (and lore). That's a **spoiler**, I can read about equipment I didn't know even existed, complete with pictures and specifications...
The whole "who are those guys" part is gone.  :-(

I admit this would be big undertaking, but I do think that there should be two kind of captions: The current ones, for when you've been introduced into a civilization, for items you see in the local shipyard/outfitter, and a very summary one ("Unknown equipment, probably a gun.") **for new civilizations you haven't contacted yet (have not yet landed on one of their planets, _and_ visited their shops, _and_ bought a local map (at least one))**.
**Only exception**: If you capture it, one could assume this piece of equipment will be thoroughly examined and is henceforth "known". But only _this_ piece of equipment, not the whole civilization.

Same applies for author ships. Alien (i.e. not commonly available) equipment should always get the "unknown equipment" caption. At best with a qualifier ("Unknown equipment, probably a gun. Looks very powerful.")
